bookwyrm/migrations/0033_reviewrate.py

```diff
@@ -0,0 +1,72 @@
+# Generated by Django 3.0.7 on 2021-01-05 05:32
+
+from django.db.models.fields.reverse_related import ManyToOneRel
+from django.db.models import Q
+from django.db import migrations, models
+import django.db.models.deletion
+
+def convert_review_rating(app_registry, schema_editor):
```

Comment:
So, since ReviewRating is a subclass of Review and thus the database implementation is literally just inserting a pointer to the relevant `review` in `bookwyrm_reviewratings`, I think if we wanted to get fiddly with it we could basically just `INSERT INTO bookwyrm_reviewrating(review_ptr_id) SELECT id FROM bookwyrm_review` and directly convert those reviews into reviewratings.

I'm not a huge fan of the direct database modifications, but when it's a simple database operation and the alternative is also gnarly...blah. I poked around and found a 13-year-old(!) ticket asking for the ability to do this properly in Django (create a child model from an existing parent model) and it is still open  https://code.djangoproject.com/ticket/7623

Reply:
oh I love this, it really is gnarly in a wholly different way. and for this specific case, yeah, it seems like a sensible fit